riscv-page-tables/src/page_table.rs

```diff
@@ -585,6 +600,64 @@ impl<T: PagingMode> PageTableInner<T> {
         entry
     }
 
+    /// Creates contiguous mappings, both physically and virtually, of `num_pages` pages, starting
+    /// from `start_vaddr` and `start_paddr`, with the same permissions.
+    ///
+    /// # Safety
+    ///
+    /// The caller must guarantee that app pages reference from `start_paddr` for `num_pages` are
+    /// uniquely owned by the root `GuestStagePageTable`.
+    #[allow(dead_code)]
```

Comment:
This doesn't need to be dead code since it's called from `map_contiguous`.

Reply:
Very nit and very true. Added that mid-commit to test then more code got added.

Reply:
fixed.